functions/__fzf_search_current_dir.fish

```diff
@@ -6,11 +6,12 @@ function __fzf_search_current_dir --description "Search the current directory. R
     set fd_arguments --hidden --color=always --exclude=.git
     set fzf_arguments --multi --ansi
     set token (commandline --current-token | string unescape)
+    set expanded_token (string replace --regex -- "^~/" $HOME/ $token)
```

Comment:
In fact we can use a unquoted `~` to replace `$HOME`, but that would be too confusing 😄

Reply:
I personally find `string replace --regex -- "^~/" ~/ $token` more aesthetically pleasing, but this works too.